packages/web-components/fast-components-msft/.storybook/main.js

```diff
@@ -4,14 +4,15 @@ module.exports = {
     stories: ["../src/**/*.stories.ts"],
     webpackFinal: async config => {
         config.module.rules.push({
-            test: /\.(ts)$/,
+            test: /\.(|ts|tsx)$/,
```

Comment:
is tsx needed?

Reply:
Ii think we can remove that. It's a vestigial copy paste oops. I take care of that real quick.